docs/simulators.rst

```diff
@@ -16,6 +16,36 @@ See the individual entries for details on each interface's capabilities and how
 Currently Supported
 ===================
 
+MetaDrive (Optional)
+----------------------------
+
+Scenic supports integration with the `MetaDrive <https://metadriverse.github.io/metadrive/>`_ simulator as an optional dependency,
+enabling users to describe dynamic simulations of vehicles, pedestrians, and traffic scenarios.
+If your system supports it, you can install it with:
+
+.. code-block:: console
+
+    pip install scenic[metadrive]
+
+Scenic supports both 2D and 3D rendering modes for MetaDrive simulations.
+2D rendering is available on all systems, providing a top-down view.
+However, 3D rendering may not work properly on macOS devices with M-series chips.
+Additionally, there is an issue where cars do not fully brake in certain scenarios.
+These issues are expected to be addressed in the next version of MetaDrive.
+
+Scenic uses OpenDRIVE maps, while MetaDrive relies on SUMO maps. Scenic provides corresponding SUMO maps for OpenDRIVE maps under the :file:`assets/maps/CARLA` directory.
+Additionally, you can convert your own OpenDRIVE maps to SUMO maps using the `netconvert <https://sumo.dlr.de/docs/Networks/Import/OpenDRIVE.html/>`_ tool.
+To avoid setting the SUMO map manually, name it the same as your OpenDRIVE file and place it in the same directory.
+Otherwise, specify it explicitly with:
+
+.. code-block:: console
+
+    --param sumo_map your_map.net.xml
```

Comment:
People don't necessarily need to do this on the command line: they could also set the parameter in their Scenic script. I'd just say something like "Otherwise, you can specify it explicitly using the `sumo_map` global parameter."